tokio-console/src/config.rs

```diff
@@ -167,7 +218,9 @@ impl Config {
 
 impl ViewOptions {
     pub fn is_utf8(&self) -> bool {
-        self.lang.ends_with("UTF-8") && !self.ascii_only
+        let lang = self.lang.as_deref().unwrap_or_default();
+        let ascii_only = self.ascii_only.unwrap_or(true);
+        lang.ends_with("UTF-8") && !ascii_only
```

Comment:
nit: it occurs to me that we can simplify this a bit if we check the value of `self.ascii_only` _before_ we touch `self.lang`:
```suggestion
        if !self.ascii_only.unwrap_or(true) {
            return false;
        }
        self.lang.as_deref().unwrap_or_default().ends_with("UTF-8") 
```
this has the advantage of avoiding the string comparison when the `ascii_only` flag is set.